src/NBitcoin/Network.cs

```diff
@@ -107,14 +107,9 @@ public abstract class Network
         public int DefaultPort { get; protected set; }
```

Comment:
Thought about putting DefaultPort and RPCPort in NodeDefaults but decided against - it seems they're not really defaults but actual network settings, aka you can't really change them, they're used too prolifically